src/humancellatlas/data/metadata/__init__.py

```diff
@@ -582,10 +582,9 @@ def specimens(self) -> List[SpecimenFromOrganism]:
         return [s for s in self.biomaterials.values() if isinstance(s, SpecimenFromOrganism)]
 
     @property
-    def sequencing_input(self) -> List[CellSuspension]:
+    def sequencing_input(self) -> List[Biomaterial]:
         return [bm for bm in self.biomaterials.values()
-                if isinstance(bm, CellSuspension)
-                and any(ps.is_sequencing_process() for ps in bm.to_processes.values())]
+                if any(ps.is_sequencing_process() for ps in bm.to_processes.values())]
```

Comment:
If I'm understanding the difference between this PR and PR #17, this one restricts the match to a `is_sequencing_process` while PR #17 restricts to `SequenceFile` outputs. I think matching on the process is preferred to matching on the output file, as the _process_ - not the output file - is what defines the "assay". If my interpretation is correct, I prefer PR #14. @tburdett ?